lib/closex/client_behaviour.ex

```diff
@@ -19,6 +19,7 @@ defmodule Closex.ClientBehaviour do
   @callback get_opportunity(id, opts) :: result
   @callback create_opportunity(map, opts) :: result
   @callback update_opportunity(id, map, opts) :: result
+  @callback create_task(id, String.t(), map, opts) :: result
```

Comment:
ah, did you put `@impl true` on the implementation I cant remember

Reply:
Nope! Thank you.